frontend/src/components/studio/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'common/types/types';
+import { Icon } from '../../common/icon';
+import { IconName, IconColor, AppRoute } from '../../../common/enums/enums';
+import { Link } from '../../common/common';
+import { useState } from 'react';
+import { useNavigate } from 'hooks/hooks';
+
+import styles from './styles.module.scss';
+
+type Props = Record<string, unknown>;
+
+const arrayButtons = [IconName.HOME, IconName.ANALYTICS];
+
+const StudioSidebar: FC<Props> = () => {
+  const navigate = useNavigate();
+  const [activeButton, setActive] = useState<string>(IconName.HOME);
+  const getColor = (item: string): string => (activeButton === item ? IconColor.GREEN : IconColor.WHITE);
+
+  const handleClick = (item: string): void => {
+    setActive(item);
+    navigate(`${item === IconName.HOME ? AppRoute.STUDIO : AppRoute.ANALYTICS}`);
```

Comment:
Looks wrong. How it would work if we decide to add more pages in the sidebar?

You need to prepare a config and then map it to buttons.

like this:

```javascript
// Create an addition file config.ts near your component that can 
export const sideBarItems: SideBarItem = [];
```

Possible type for SIdeBarItem:

```javascript
export type SideBarItem = {
  routeName: ...; // Enum
  iconType: ...;
  itemName: ...; // Display name for link
}
```